stock_inventory_hierarchical_location/__init__.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
```

Comment:
It will be done in another PR when we migrate this module (please cf. my reply regarding  `stock_inventory_hierarchical_location/__openerp__.py`)